scripts/ci/run-tests

```diff
@@ -36,6 +36,8 @@ def process_args(args):
         test_args += "-n auto --dist=loadfile --maxprocesses=4 "
     if args.with_cov:
         test_args += f"--cov={PACKAGE} --cov-report xml "
+    if args.zipped:
+        test_args += "--import-mode=append "
```

Comment:
this is required for the test runner to check paths in the correct order for loading modules (i.e. prioritizing `botocore.zip` over botocore on disk

Reply:
Sure. I think I'd prefer the latter since that reflects the style of the other existing flags